example/src/tests/groupTests.ts

```diff
@@ -1247,6 +1247,35 @@ test('can stream all groups and conversations', async () => {
   return true
 })
 
+test('can stream groups and messages', async () => {
+  for (const env of ['local', 'dev'] as const) {
```

Comment:
it would be the only test in that test suite that runs not only on `local` env. Is that ok?